Automattic/wp-calypso
# Theme bundling: Added woocommerce badge component

#### Proposed Changes

* Added WooCommerce badge to themes with bundled plugins.

#### Testing Instructions



- ~Apply patch D87710-code~
- Create a site with these flags on: `themes/plugin-bundling, signup/seller-upgrade-modal`
- On the design picker find Thriving Artist (that has bundling)
- Check the badge
![image](https://user-images.githubusercontent.com/3801502/193027096-7fb5b78a-7c42-4342-8d2c-04c96b379e92.png)



Related to https://github.com/Automattic/wp-calypso/issues/66654
